src/common/decorators/swagger.ts

```diff
@@ -115,22 +115,39 @@ export function ApiTransactionResponse(
   );
 }
 
+type FailedTransactionResponseCodes =
```

Comment:
I'd either remove this, or define it as `keyof typeof httpStatusDecoratorMap` so it will get updated automatically